src/Compilers/Core/AnalyzerDriver/AnalyzerManager.cs

```diff
@@ -301,6 +301,20 @@ internal bool IsSupportedDiagnostic(DiagnosticAnalyzer analyzer, Diagnostic diag
 
             foreach (var diag in supportedDiagnostics)
             {
+                if (HasNotConfigurableTag(diag.CustomTags))
```

Comment:
@shyamnamboodiripad This code lies in the shared AnalyzerDriver project, and it cannot access the internal DiagnosticDescriptor.IsNotConfigurable method when compiled into the Features project.